parse_expr_data.R

```diff
@@ -1,77 +1,69 @@
 #!/usr/bin/env Rscript
 
-# Load optparse we need to check inputs
 suppressPackageStartupMessages(require(optparse))
-# Load common functions
 suppressPackageStartupMessages(require(workflowscriptscommon))
+suppressPackageStartupMessages(require(garnett))
```

Comment:
Please move the package loads (except for optparse) back to after argument parsing and checking. No point spending time doing the load until we've done those checks.